src/query/validation.c

```diff
@@ -209,7 +216,7 @@ static bool verify_aggregator(Node *node, void *context)
 
 static void verify_aggregators(Query *query)
 {
-  query_tree_walker(query, verify_aggregator, NULL, 0);
+  query_tree_walker(query, verify_aggregator, query, 0);
```

Comment:
is this passing `query` as context?

Reply:
Yes, needed for `verify_count_histogram`.

Reply:
ah, I'm blind, I looked for this but missed.